src/components/DataTable/index.jsx

```diff
@@ -159,48 +159,53 @@ const DataTable = ({
   }, [filters]);
 
   return (
-    <div {...getTableProps()} className={className}>
+    <div {...getTableProps()} className={className} tabIndex="0">
       <div className={tableContainerClassName}>
-        {headerGroups.map((headerGroup) => (
-          <div
-            {...headerGroup.getHeaderGroupProps()}
-            className={headerGroupClassName}
-          >
-            {headerGroup.headers.map((column) => (
-              <>
-                <div
-                  className={headerCellClassName}
-                  {...column.getHeaderProps(
-                    columnSort ? column.getSortByToggleProps() : undefined
-                  )}
-                >
-                  <span className={headerCellTextClassName}>
-                    {column.render("Header")}
-                  </span>
-                  {columnSort && (
-                    <span
-                      className={
-                        column.isSorted
-                          ? column.isSortedDesc
-                            ? columnIsSortedDecClassName
-                            : columnIsSortedAscClassName
-                          : columnIsSortedClassName
-                      }
-                    />
-                  )}
-                </div>
-                {columnResize && (
-                  <div
-                    {...column.getResizerProps()}
-                    className={`${tableColumnResizer} ${
-                      column.isResizing ? tableColumnIsResizing : ""
-                    }`}
-                  />
-                )}
-              </>
-            ))}
-          </div>
-        ))}
+        {headerGroups.map((headerGroup) => {
+          const { key } = headerGroup.getHeaderGroupProps()
+          return(
+            <div key={key}
+              {...headerGroup.getHeaderGroupProps()}
+              className={headerGroupClassName}
+            >
+              {headerGroup.headers.map((column) => {
+                return(
+                  <div key={column.id}>
+                    <div
+                      className={headerCellClassName}
+                      {...column.getHeaderProps(
+                        columnSort ? column.getSortByToggleProps((props) => ({...props, title: `Sort by ${column.Header}`})) : undefined
```

Comment:
Currently all table header titles just read as `Sort by` when hovered with a mouse. This update will change it to `Sort by my header name`. Which should clear confusion when looking at all the headers.